.github/workflows/bild.yml

```diff
@@ -0,0 +1,27 @@
+name: bild
```

Comment:
Rename the file to build.yml.

Reply:
I initiated the change and resolved the conflict